src/backend/distributed/utils/citus_ruleutils.c

```diff
@@ -337,7 +348,15 @@ pg_get_tableschemadef_string(Oid tableRelationId, bool includeSequenceDefaults)
 	{
 		Form_pg_attribute attributeForm = tupleDescriptor->attrs[attributeIndex];
 
-		if (!attributeForm->attisdropped && attributeForm->attinhcount == 0)
```

Comment:
I think it would be good to avoid confusing anyone when we announce partitioning support.